src/isar_turtlebot/turtlebot/turtlebot.py

```diff
@@ -50,67 +37,28 @@ def __init__(self, bridge: RosBridge) -> None:
         self.task_handler: Optional[TaskHandler] = None
 
         self.filenames: dict = dict()
+        self.metadata: dict = dict()
```

Comment:
```suggestion
        self.inspections: dict = dict()
```
Could this be renamed to `inspections`, as it stores the inspection instances returned from `task_handler.get_inspection()`?